File: competition/05-dict/dict.js
const counts = new Map();
const buckets = 20000;
for (let i = 0; i < 200000; i++) {
  const key = String(i % buckets);
  counts.set(key, (counts.get(key) || 0) + 1);
}
console.log(counts.size);
console.log(counts.get("0") || 0);

File: competition/05-dict/dict.lua
-- Word-frequency tally over a stream of keys — hash table insert/lookup.
local counts = {}
local buckets = 20000
for i = 0, 199999 do
  local key = tostring(i % buckets)
  counts[key] = (counts[key] or 0) + 1
end
local size = 0
for _ in pairs(counts) do size = size + 1 end
print(size)
print(counts["0"] or 0)
